src/chapters/errors-and-debugging/how-to-avoid-debugging.rst

```diff
@@ -0,0 +1,22 @@
+How to Avoid Debugging
+======================
+
+While debugging is an unavoidable part of programming, you can reduce the number of bugs you encounter by working carefully.
+
+Start Small
+-----------
+
+This is probably the single biggest piece of advice for programmers at every level. It can be tempting to sit down and try to write an entire program at once. But when the program does not work, then you have a large number of possibilities for where things might have gone wrong. The more code, the more possibilities exist. Where to start? Where to look first? How to figure out what went wrong? 
+
+When you start work on a large program, it is best to break it down into smaller steps and then start coding just one very small part of it. You might write just 2 lines of code and then make sure that the program runs properly. 
+
+Regularly running your code is quick and easy, and gives you immediate feedback about whether what you have just done is okay or not. 
```

Comment:
Remove "or not".

Reply:
I would actually reword the sentence. "Regularly running your code is quick, easy, and gives you immediate feedback about how the code will run."